src/inspector/models/text/textsettingsmodel.cpp

```diff
@@ -116,6 +117,8 @@ void TextSettingsModel::loadProperties()
 
     m_fontSize->setIsEnabled(true);
 
+    loadPropertyItem(m_textLineSpacing);
```

Comment:
I realised that we need to add `formatDoubleFunc` here, like `loadPropertyItem(m_textLineSpacing, formatDoubleFunc);`. That is to round the value before it is shown in the UI. Without it, we might get strange things like 9 decimals :)
(and yes, this is necessary even though we already have `decimals: 2` in QML.)